### DISCUSSION TITLE
Engine Compatibility Question

### DISCUSSION THREAD

**Discussion by StevenWine** [The-Wineskin-Project/WineskinServer#361] [Q&A]
HI there, I'm trying to setup a wine for Everquest Titanium. I've found a new guide that says to use engine WS9Wine2.2, but this engine is no longer available. Is there a compatible engine i could use in its place? I'm not super savvy on how all this works, but just trying to follow a how-to guide. Thanks!

**Comment by Gcenx** [comment]
You’d need to ask the author of the guide as I’ve dropped support for anything below wine-5.0, another thing to keep in mind is only WS11 Engines will function on macOS Catalina and later.

WS9 Engines require macOS High Sierra or macOS Mojave, technically Wineskin can support below wine-5.0 but it does require an installation of XQuartz though winetricks no longer supports these ancient wine veraions.
